guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/config/ConfigurationServiceImpl.java

```diff
@@ -74,32 +74,23 @@
     // mainly for windows as *NIX is based on POSIX but escape always to keep it consistent
     private static final String INVALID_FILENAME_CHARS = "[\\,/,:,*,?,\",<,>,|]";
 
-    @Inject
-    @Named("system")
     private org.guvnor.structure.repositories.Repository systemRepository;
 
-    @Inject
     private ConfigGroupMarshaller marshaller;
 
-    @Inject
     private User identity;
 
     //Cache of ConfigGroups to avoid reloading them from file
     private final Map<ConfigType, List<ConfigGroup>> configuration = new ConcurrentHashMap<ConfigType, List<ConfigGroup>>();
     private AtomicLong localLastModifiedValue = new AtomicLong( -1 );
 
-    @Inject
-    @Named("configIO")
     private IOService ioService;
 
     // monitor capabilities
-    @Inject
     @Repository
```

Comment:
You'll need to move these qualifiers to the constructor; e.g. ```@OrgUnit Event<SystemRepositoryChangedEvent> orgUnitChangedEvent``` and delete them here.